app/models/order.rb

```diff
@@ -8,7 +8,12 @@ class Order < ApplicationRecord
   validates :pay_type, inclusion: pay_types.keys
 
   has_many :line_items, dependent: :destroy
+  belongs_to :user, optional: true
 
+  scope :by_date, ->(from_date = nil, to_date = nil) do
```

Comment:
Not a good way of doing it
use `Time.current.beginning_of_day` and `Time.current.end_of_day`